### DISCUSSION TITLE
single chapter from multiple files in _quarto.yml?

### DISCUSSION THREAD

**Discussion by actsasgeek** [quarto-dev/quarto-cli#1789] [Q&A]
I can see we can make parts from multiple chapters but is there a way to make a single chapter from multiple files? Jupyter notebooks often get unwieldy so split chapters into separate files.

thanks!

----- 
**Update**

I have a sort of workaround. I used includes. However, this is less than ideal for a few reasons.

1. We're back to the monolithic file. Although it's not a file I have work with, I think it is creating a single file to execute.
2. Changes to the included files are not detected so I have to do weird things to the index.qmd file to get it to "recompile".
3. The chapter hierarchy is not handled automatically. so all the "#" in the included files need to be changed to "##", etc. This isn't great when the files are viewed individually (the heading hierarchy is mucked up).

**Comment by jjallaire** [comment]
There currently isn't (although with `.qmd` you can use the `{{< include >}}` shortcode to compose a chapter from multiple sub-files.

**Reply by actsasgeek** [reply]
Thank you for getting back to me. I'll work something out for now. I would give a big thumbs up to this as a feature, although I know it's probably a headache. I've outlined the issues with using includes above. Thanks again for a great tool.
